vault/dendron.tutorial.md

```diff
@@ -1,16 +1,18 @@
 ---
 id: 678c77d9-ef2c-4537-97b5-64556d6337f1
-title: Dendron Tutorial
+title: Getting Started
 desc: ''
-updated: 1620514407615
+updated: 1620596651093
 created: 1608051264282
 nav_order: 1.1
 ---
-- NOTE: if you'd rather follow a video tutorial, fellow dendrologist [Ian Jones](https://github.com/theianjones) made a fantastic video series which you can follow [here](https://egghead.io/courses/build-a-personal-knowledge-management-system-with-dendron-b24b)
+
 
 ## Pre-requisites
 
-This walkthrough will help you get started with Dendron. Before you start, make sure you have Dendron installed. If not, you can do so by following the instructions [[here|dendron.guides.install]].
+This walkthrough will help you get started with Dendron. Before you start, make sure you have Dendron installed and a workspace initialized. 
```

Comment:
I think this should be emphasized even more. I'd go as far as to say it should have a colored callout with an emoji in front of the text 😄  maybe bold / italic on  `workspace initialized` for now? 
```suggestion
This walkthrough will help you get started with Dendron. Before you start, make sure you have Dendron installed and **_a workspace initialized_**. 
```